### DISCUSSION TITLE
Icons Not Rendering In VS Code Win 10

### DISCUSSION THREAD

**Discussion by jpvonhemel** [JanDeDobbeleer/oh-my-posh#2161] [Troubleshoot]
Hello,

I am using VS Code and have CaskaydiaCove NF set as my editor Font for the text editor.   I am seeing the colors render, but not the unicode icons.   I noticed on Scott Hanselman's WT, he sees the diamons/triangles in the json file inside the editor.   I read the GPU is required and made sure this was set to the default of auto.   I do see nerd font icons inside powershell.   I updated Oh My Posh to the latest version and I am using Jan's latest json file.   I figure I have done something in my fiddling to have caused this, but have been unable to figure out just what that is.   I am attaching a screen shot.   Any ideas?

Thank you,

Jerold

![Screenshot 2022-04-26 092132](https://user-images.githubusercontent.com/20939267/165335360-47f18430-0250-4dcd-a306-ef15ffeac8b3.png)
https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by JanDeDobbeleer** [comment]
Hey, due to a recent change in vscode, you need to set `"terminal.integrated.minimumContrastRatio": 1` in your vscode settings. If that doesn't fix all issues, let's take it from there.

**Comment by jpvonhemel** [comment]
Thank you.  I did mean VSCode.   I made the update to minimumContrastRatio to 1, and restarted, but the issue persists.   Not sure what I have done.   

![image](https://user-images.githubusercontent.com/20939267/165341942-73998099-6ee8-4433-a4bf-1a72bee1cd4c.png)

**Comment by JanDeDobbeleer** [comment]
@jpvonhemel maybe I misunderstood, you're looking for a way to display the actual icon when looking at the json definition in the editor? That's not possible. We deliberately set these to their unicode value rather than the symbol (otherwise it's challenging understanding which icon it needs to be).
